src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/builtin.ts

```diff
@@ -384,7 +384,7 @@ export const comparisonFunctions: FunctionDefinition[] = [
   },
 ].map((op): FunctionDefinition => createComparisonDefinition(op));
 
-const likeFunctions: FunctionDefinition[] = [
+const notLikeFunctions: FunctionDefinition[] = [
```

Comment:
I dont understand this change, the not_like is not being returned from ES?

Reply:
Yes, I left this notLikeFunctions here cause I didn't see an equivalent from Elasticsearch.